packages/jest-types/src/Config.ts

```diff
@@ -171,6 +171,7 @@ export type InitialOptions = Partial<{
   globalTeardown: string | null | undefined;
   haste: HasteConfig;
   injectGlobals: boolean;
+  inlineSnapshotFormatter: 'exposed prototypes' | 'simple';
```

Comment:
instead of magic strings, should we just expose `pretty-format`'s options? `inlineSnapshotSerializationOptions` or some such?

Reply:
Yeah, this seems like a better idea - can also offer that for `snapshotFormatter` too